src/validation/__tests__/ProhibitIntrospectionQueriesRule-test.js

```diff
@@ -0,0 +1,146 @@
+// @flow strict
+
+import { describe, it } from 'mocha';
+
+import { ProhibitIntrospectionQueriesRule } from '../rules/optional/ProhibitIntrospectionQueriesRule';
+
+import {
+  GraphQLSchema,
+  GraphQLObjectType,
+  GraphQLString,
+  __Schema,
+} from '../../type/index';
+
+import { expectValidationErrorsWithSchema } from './harness';
+
+const schema = new GraphQLSchema({
+  query: new GraphQLObjectType({
+    name: 'Query',
+    fields: {
+      someQuery: {
+        type: new GraphQLObjectType({
+          name: 'SomeType',
+          fields: {
+            someField: { type: GraphQLString },
+            introspectionField: { type: __Schema },
```

Comment:
If you like to test a non-standard introspection field it's better to use some type that is not exposed directly by GraphQL itself, for example, type for enum value.